WcaOnRails/app/views/shared/_associated_events_picker.html.erb

```diff
@@ -18,9 +18,10 @@ Variables:
      associated_events.find_by_event_id(event.id) || associated_events.build(event_id: event.id)
    end %>
 
-<div class="form-group">
-  <%= label_tag events_association_name, label, class: "associated-events-label" %>
-  <div id="<%= events_association_name %>" class="associated-events">
+<% errors = form_builder.object.errors.messages[events_association_name] || [] %>
+<div class="form-group <%= "has-error" if errors.length > 0 %>">
+  <%= label_tag events_association_name, label, class: "control-label col-sm-2" %>
+  <div id="<%= events_association_name %>" class="col-sm-9">
```

Comment:
You've added these `col-sm-x` classes directly here but this input isn't always inside a horizontal form. Does it look well on the user profile edit page? Regardless the answer, I consider extending the classes only when it's a part of a horizontal form (as before) to be a bit better.